tests/installation/bootloader_svirt.pm

```diff
@@ -258,19 +266,41 @@ sub run() {
         $svirt->resume;
         wait_serial("Press enter to boot the selected OS", 10) || die "Can't get to Grub";
         type_string "echo e > \$pty\n";    # edit
-        if (is_jeos) {
+
+        if (is_jeos or check_var('DISTRI', 'casp')) {
             for (1 .. 4) { type_string "echo -en '\\033[B' > \$pty\n"; }    # four-times key down
-            type_string "echo -en '\\033[K' > \$pty\n";                     # end of line
-            type_string "echo -en ' $cmdline' > \$pty\n";
-            type_string "echo -en ' xenfb.video=4,1024,768' > \$pty\n";     # set kernel framebuffer
         }
         else {
             for (1 .. 2) { type_string "echo -en '\\033[B' > \$pty\n"; }    # four-times key down
-            type_string "echo -en '\\033[K' > \$pty\n";                     # end of line
-            type_string "echo -en ' $cmdline' > \$pty\n";
+        }
+        type_string "echo -en '\\033[K' > \$pty\n";                         # end of line
+        type_string "echo -en ' $cmdline' > \$pty\n";
+        if (sle_version_at_least('12-SP2') or check_var('DISTRI', 'casp')) {
             type_string "echo -en ' xen-fbfront.video=32,1024,768' > \$pty\n";    # set kernel framebuffer
         }
-        type_string "echo -en '\\x18' > \$pty\n";                                 # send Ctrl-x to boot guest kernel
+        else {
+            type_string "echo -en ' xenfb.video=4,1024,768' > \$pty\n";           # set kernel framebuffer
+        }
+
+        #        if (is_jeos) {
+        #            for (1 .. 4) { type_string "echo -en '\\033[B' > \$pty\n"; }    # four-times key down
+        #            type_string "echo -en '\\033[K' > \$pty\n";                     # end of line
+        #            type_string "echo -en ' $cmdline' > \$pty\n";
+        #            type_string "echo -en ' xenfb.video=4,1024,768' > \$pty\n";     # set kernel framebuffer
+        #        } elsif (check_var('DISTRI', 'casp')) {
+        #            for (1 .. 4) { type_string "echo -en '\\033[B' > \$pty\n"; }    # four-times key down
+        #            type_string "echo -en '\\033[K' > \$pty\n";                     # end of line
+        #            type_string "echo -en ' $cmdline' > \$pty\n";
+        #            type_string "echo -en ' xen-fbfront.video=32,1024,768' > \$pty\n";    # set kernel framebuffer
+        #        }
+        #        else {
+        #            for (1 .. 2) { type_string "echo -en '\\033[B' > \$pty\n"; }    # four-times key down
+        #            type_string "echo -en '\\033[K' > \$pty\n";                     # end of line
+        #            type_string "echo -en ' $cmdline' > \$pty\n";
+        #            type_string "echo -en ' xen-fbfront.video=32,1024,768' > \$pty\n";    # set kernel framebuffer
+        #        }
```

Comment:
Don't add disabled code as nobody but you will understand why this is not active. Better leave it out and keep in your local git repository, either in `git stash` or a temporary "WIP"-commit